packages/server/api/routes/favorites.router.js

```diff
@@ -95,8 +96,9 @@ router.post('/', (req, res, next) => {
  */
 
 router.delete('/:id', (req, res, next) => {
+  const { token } = req.headers;
   favoritesController
-    .deleteFavorites(req.params.id, req)
+    .deleteFavorites(token, req.params.id, req)
```

Comment:
The `req` parameter is not used by the function, you only use the first two in the function, so I would just remove it.

Reply:
Done ✅